lib/lib_install.php

```diff
@@ -90,7 +90,8 @@ function initDb() {
 	if ($db['type'] !== 'sqlite') {
 		Minz_ModelPdo::$usesSharedPdo = false;
 		$dbBase = isset($db['base']) ? $db['base'] : '';
-		$db['base'] = '';
+		//Use default database for PostgreSQL, empty database for MySQL / MariaDB:
+		$db['base'] = $db['type'] === 'pgsql' ? 'postgres' : '';
```

Comment:
This will have no effect: `$db['base']` is set line 97 with the value from line 92. Also, `initDb` is called after checking the `base` parameter is not empty ([here](https://github.com/FreshRSS/FreshRSS/blob/master/app/install.php#L117)) so I don't understand how the value can be empty.

I think we should be explicit instead of setting the value in the back of the user: what do you think of prefilling the `base` input with value `postgres` if the selected database is pgsql?

Reply:
This does have an effect on the targetted line, which is `$databaseDAO = FreshRSS_Factory::createDatabaseDAO();`
It is not in the back of the user: using `postgres` as the database name is the recommended (as far as I know) approach to create a new PostgreSQL database.
I will add more comments to clarify this section.
What we *could* do, is to add an option to allow users to autocreate the database or not. But I am not sure it is a good idea, as it will add confusion and noise in the user interface, and I better like testing whether the database exists or not (what we already do).
But what we *should* do, is to test what happens with different access rights, as I am not sure we are catching all errors properly.

Reply:
https://github.com/FreshRSS/FreshRSS/pull/2735/commits/ac5dcc98a7f300e799f77a55f024efcaac5c1ac7

Reply:
Ah, I think I now understand your suggestion. No, we should not use the default `postgres` database for our own data.

Reply:
Ah, I see. Maybe a better solution would be to not use the `base` argument from the configuration when creating the database instead of switching the value here

Reply:
This could probably be refactored, but maybe that this a job for another PR :-)

Reply:
Of course ;) and for another time too!

Reply:
Any remaining comment then? :-)

Reply:
Nope! (I don't know why, I thought it was already merged :))